engine/src/flutter/impeller/entity/shaders/line.frag

```diff
@@ -42,5 +42,7 @@ float CalculateLine() {
 void main() {
   float line = CalculateLine();
   frag_color = vec4(frag_info.color.xyz, line);
+  // frag_color =
```

Comment:
commented out code

Reply:
I have rewritten this so many times I decided to keep it in.  I added more to the comment to explain what it is and why it's there.

Reply:
SGTM